src/table/table.scss

```diff
@@ -4,383 +4,423 @@
 @import '../icon/index';
 @import './variables';
 
-@mixin iui-tables-table {
+@mixin iui-table {
   @include iui-reset;
-
   @include iui-font-family;
   display: flex;
   flex-direction: column;
-  padding: 0;
 
   * {
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
-    width: 100%;
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
+  .iui-table-header {
+    @include iui-table-header;
   }
 
-  &.iui-zebra-striping > .iui-tables-body > .iui-tables-row:nth-child(even) {
+  .iui-table-body {
+    @include iui-table-body;
+  }
+
+  .iui-cell {
+    @include iui-table-cell;
+  }
+
+  &.iui-zebra-striping > .iui-table-body > .iui-row:nth-child(even):not(.iui-selected) {
     @include themed {
       background: rgba(t(iui-color-foreground-body-rgb), 0.02);
     }
   }
 
-  &.iui-tables-fixed > .iui-tables-body {
-    max-height: 300px;
+  &.iui-fixed > .iui-table-body {
+    max-height: $iui-baseline * 27;
     overflow-x: hidden;
     overflow-y: scroll;
     overflow: overlay;
   }
+}
 
-  .iui-select .iui-select-button {
-    background-color: transparent;
-    border: none;
+@mixin iui-table-header { 
+  user-select: none;
```

Comment:
This is needed, right?
I might want to copy header column value, but... maybe not valid..

Reply:
The reason I added this was to avoid accidental highlighting of the column header text when the user clicks to sort the column.